Source/Core/Core/IOS/ES/Formats.h

```diff
@@ -219,8 +219,8 @@ class UIDSys final
 public:
   explicit UIDSys(Common::FromWhichRoot root);
 
-  u32 GetUIDFromTitle(u64 title_id);
-  void AddTitle(u64 title_id);
+  u32 GetUIDFromTitle(u64 title_id) const;
+  u32 AddTitle(u64 title_id);
```

Comment:
Does this function return the uid? That's a bit confusing IMO. If it's how the function is implemented in IOS, it can't be helped. Otherwise I don't think that's a good idea. I won't be against a boolean for success/failure, though.